src/openttd.cpp

```diff
@@ -330,6 +330,24 @@ static void ShutdownGame()
 	UninitFreeType();
 }
 
+/**
```

Comment:
Simplify in a single line:

/** Get the introduction game landscape specific save file */